digest/digester_resumable.go

```diff
@@ -0,0 +1,45 @@
+// +build resumabledigest
```

Comment:
Let's call the flag "noresumabledigest", so default is to build it with resumable.